course/assets/css/video.scss

```diff
@@ -128,3 +128,21 @@ a.video-download-button:visited {
     padding-bottom: 0px;
   }
 }
+
+.video-tab-header {
+  background-color: #f6f7f9;
+  font-size: $font-lg;
+  padding-left: 15px;
+  padding-right: 15px;
+  padding-bottom: 5px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+}
+
+.video-tab-section-toggle > div > div > i:after {
```

Comment:
I think a better way to do this would be
`
.video-tab-section-toggle {
  i:after {
    content: "add";
  }
}
`